cloudigrade/api/clouds/aws/util.py

```diff
@@ -339,7 +339,7 @@ def save_instance(account, instance_data, region):
             # We are not sure how this could happen. Whenever we save a new
             # AwsMachineImage, we *should* always follow up with creating
             # its paired MachineImage. Investigate if you see this error!
-            logger.error(
+            logger.info(
                 _(
                     "Existing AwsMachineImage %(awsmachineimage_id)s "
                     "(ec2_ami_id=%(ec2_ami_id)s) found has no "
```

Comment:
We should drop the `This should not happen!` from the line below, as this happens all the time :)

Reply:
Thank you!